packages/lib-user/src/components/UserHome/components/Dashboard/DashboardContainer.js

```diff
@@ -0,0 +1,35 @@
+import { panoptes } from '@zooniverse/panoptes-js'
+import auth from 'panoptes-client/lib/auth'
+import useSWR from 'swr'
+
+import Dashboard from './Dashboard'
+
+const fetchProfileBanner = async ({ authUser }) => {
+  const token = await auth.checkBearerToken()
+  const authorization = `Bearer ${token}`
+  const query = { include: 'profile_header', user_id: authUser.id }
+
+  try {
+    const { body } = await panoptes.get('/users', query, { authorization })
+    const profileBannerSrc = body.linked.profile_headers[0].src
+    return profileBannerSrc
+  } catch (error) {
+    console.error(error)
+    return null
+  }
+}
+
+const SWROptions = {
+  revalidateIfStale: true,
+  revalidateOnMount: true,
+  revalidateOnFocus: true,
+  revalidateOnReconnect: true,
+  refreshInterval: 0
+}
+
+export default function DashboardContainer({ authUser }) {
+  const key = { authUser }
+  const { data: profileBannerSrc } = useSWR(key, fetchProfileBanner, SWROptions)
```

Comment:
[minor] Re: profile avatar & profile banners aren't displaying properly.

> There's something happening when running app-root locally where a user's avatar and profile banner image aren't consistently displayed 🤔 I'm going to investigate tomorrow before I complete the rest of the Dashboard (draft PR https://github.com/zooniverse/front-end-monorepo/pull/6121)

I'm wondering if setting the key as an authUser _object,_ instead of say authUser_id _string,_ could be affecting the banner fetch.
- I only _just_ learnt that useSWR accepts objects as a key [(docs)](https://swr.vercel.app/docs/arguments) instead of just strings, so I'm not familiar with how reliable the old-key-vs-new-key comparison would be to trigger/not trigger a refresh.
- in any case, fetchProfileBanner() seems to only need the User ID, so using that as the string-key here could narrow down the potential problem triggers by one.

That said, I really can't comment more until I pull this branch to local and see if I can replicate the issue. Time to spin up some test accounts with banners and without banners!

Reply:
I think Jim and I figured out the cause was `avatar_src` not being properly included with the user object passed from app-root. I've refactored this PR accordingly, and it's ready for review! Please feel free to assign yourself to review, but no worries if not, I can rope Mark in next week.